packages/grid/src/Grid.css

```diff
@@ -109,43 +46,48 @@
-  --grid-editableCell-borderColor-hover: var(--uitkGrid-editableCell-borderColor-hover, var(--grid-editableCell-borderColor-hover-default));
-  --grid-editableCell-background-active: var(--uitkGrid-editableCell-background-active, var(--grid-editableCell-background-active-default));
-  --grid-editableCell-color-active: var(--uitkGrid-editableCell-color-active, var(--grid-editableCell-color-active-default));
-  --grid-editableCell-cornerTag-size: var(--uitkGrid-editableCell-cornerTag-size, 8px);
+  --grid-row-borderColor-selected: var(--uitkGrid-row-borderColor-selected, var(--uitk-selectable-borderColor-selected));
+  --grid-cell-background-selected: var(--uitkGrid-cell-background-selected, var(--uitk-selectable-background-selected));
+  --grid-editableCell-borderColor: var(--uitkGrid-editableCell-borderColor, var(--uitk-editable-borderColor));
+  --grid-editableCell-borderColor-hover: var(--uitkGrid-editableCell-borderColor-hover, var(--uitk-editable-borderColor-hover));
+  --grid-editableCell-background-active: var(--uitkGrid-editableCell-background-active, var(--uitk-editable-background-active-medium));
+  --grid-editableCell-color-active: var(--uitkGrid-editableCell-color-active, var(--uitk-text-primary-foreground));
+  --grid-editableCell-cornerTag-size: var(--uitkGrid-editableCell-cornerTag-size, var(--grid-editableCell-cornerTag-size-default));
 
-  --grid-header-color: var(--uitkGrid-header-color, var(--grid-header-color-default));
+  --grid-header-color: var(--uitkGrid-header-color, var(--uitk-text-secondary-foreground));
 
-  --grid-headerColumnSeparator-color: var(--uitkGrid-headerColumnSeparator-color, var(--grid-headerColumnSeparator-color-default));
-  --grid-headerRowSeparator-color: var(--uitkGrid-headerRowSeparator-color, var(--grid-headerRowSeparator-color-default));
+  --grid-headerColumnSeparator-color: var(--uitkGrid-headerColumnSeparator-color, var(--uitk-separable-tertiary-borderColor));
+  --grid-headerRowSeparator-color: var(--uitkGrid-headerRowSeparator-color, var(--uitk-separable-primary-borderColor));
   --grid-headerRowSeparator-gap: var(--uitkGrid-headerRowSeparator-gap, var(--uitk-size-basis-unit));
   --grid-headerRowSeparator-height: var(--uitkGrid-headerRowSeparator-height, 1px);
-  --grid-groupHeaderRowSeparator-color: var(--uitkGrid-groupHeaderRowSeparator-color, var(--grid-groupHeaderRowSeparator-color-default));
+  --grid-groupHeaderRowSeparator-color: var(--uitkGrid-groupHeaderRowSeparator-color, var(--uitk-separable-tertiary-borderColor));
+
+  --grid-groupHeader-color: var(--uitkGrid-groupHeader-color, var(--uitk-text-secondary-foreground));
+  --grid-shadow-color: var(--uitkGrid-shadow-color, var(--uitk-overlayable-shadow-scroll));
+
+  --grid-columnDropTarget-color: var(--uitkGrid-columnDropTarget-color, var(--uitk-target-borderColor-hover));
+  --grid-columnDropTarget-borderWidth: var(--uitkGrid-columnDropTarget-borderWidth, var(--uitk-target-borderWidth-hover));
```

Comment:
instead of --uitk-target-borderWidth-hover, can you use 0 var(--uitk-size-border)